# Remove the "--disable-setuid-sandbox" command line flag as a bad flag

Index: chromium-130.0.6723.44/chrome/browser/ui/startup/bad_flags_prompt.cc
===================================================================
--- chromium-130.0.6723.44.orig/chrome/browser/ui/startup/bad_flags_prompt.cc
+++ chromium-130.0.6723.44/chrome/browser/ui/startup/bad_flags_prompt.cc
@@ -75,7 +75,6 @@ const char* const kBadFlags[] = {
     // These flags disable sandbox-related security.
     sandbox::policy::switches::kDisableGpuSandbox,
     sandbox::policy::switches::kDisableSeccompFilterSandbox,
-    sandbox::policy::switches::kDisableSetuidSandbox,
     sandbox::policy::switches::kNoSandbox,
 #if BUILDFLAG(IS_WIN)
     sandbox::policy::switches::kAllowThirdPartyModules,
